cmd/cli/cli.go

```diff
@@ -93,7 +95,9 @@ func RunCli(supportedSources []config.ConfigurationPair, supportedTransformation
 		}
 	}
 
-	app.Run(os.Args)
+	if err := app.Run(os.Args); err != nil {
+		log.WithError(err).Error("failed to run cli")
```

Comment:
@oo-sleeper I might be forgetting context here - is there any reason this isn't a fatal error?